crates/swc_ecma_minifier/tests/fixture/next/43052/input.js

```diff
@@ -0,0 +1,224 @@
+(self["webpackChunk_N_E"] = self["webpackChunk_N_E"] || []).push([
```

Comment:
Can we include a more minimal example that only has a an eval, and assert the vars aren't mangled?

Reply:
I thought this was the minimal repro case, but it seems like I accidentally deleted `eval` while trying to reduce.

Reply:
I added `mangle.json` to enable mangler and added some test code to input to ensure that name mangler is applied to the test case.